### DISCUSSION TITLE
Welcome to awesome-digitalocean Discussions!

### DISCUSSION THREAD

**Discussion by jonleibowitz** [jonleibowitz/awesome-digitalocean#13] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ZackaryBob** [comment]
I am the noob of noobs when it comes to tech.  Virtually all the questions I will ask will be stupid ones, so please don't judge me.

**Comment by ZackaryBob** [comment]
First question: am I the only person to post a question on this discussion board?
